src/utils/mdx-component-mapper.js

```diff
@@ -1,19 +1,17 @@
 /* eslint-disable react/display-name */
 import * as React from 'react';
 import {
-  CodePane,
+  Heading,
```

Comment:
There were some unused imports in this file. I usually clean this up with an editor action for "optimizing imports" (which clears those out). Looks like it also sorts them by alphabetical order, which is why this diff looks bigger than it should. (In effect, all that happened was `CodePan` and `CodeSpan` were removed.)